### PR TITLE
Adding app engine request handling samples.

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

File: appengine/requests/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## App Engine Requests Docs Snippets
+
+These snippets demonstrate various aspects of App Engine Python request handling.
+
+<!-- auto-doc-link -->
```
Comment (jerjou) [inline appengine/requests/README.md]: What are these auto-doc-link comments I'm seeing?

File: appengine/requests/main.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+"""
+
+import os
+import time
+
+import webapp2
+
+
+# [START request_timer]
+class TimerHandler(webapp2.RequestHandler):
```
Comment (jerjou) [inline appengine/requests/main.py]: FYI you can do indented_block="class TimerHandler" to pull in the entire class

**Comment by jerjou** [comment]
LGTM

The region tags shouldn't be necessary, but I won't block on it.

File: appengine/requests/README.md (hunk 1 above)
Comment (jonparrott) [inline appengine/requests/README.md]: there's a script that can automatically populate these. I need to re-run it once we're done moving all samples.

**Pull request merged**